crates/iota-types/src/unit_tests/passkey_authenticator_test.rs

```diff
@@ -353,56 +352,71 @@ async fn test_passkey_fails_wrong_client_data_type() {
-    let response = PasskeyResponse::<TransactionData> {
-        user_sig_bytes: Hex::decode("02bbd02ace0bad3b32eb3a891dc5c85e56274f52695d24db41b247ec694d1531d6fe1a5bec11a8063d1eb0512e7971bfd23395c2cb8862f73049d0f78fd204c6d602276d5f3a22f3e698cdd2272a63da8bfdd9344de73312c7f7f9eca21bfc304f2e").unwrap(),
-        authenticator_data: Hex::decode("49960de5880e8c687434170f6476605b8fe4aeb9a28632c7995cf3ba831d97631d00000000").unwrap(),
-        client_data_json: r#"{"type":"webauthn.get","challenge":"AAAAZgUD1inhS1l9qUfZePaivu6IbIo_SxCGmYcfTwrmcFU","origin":"http://localhost:5173","crossOrigin":false}"#.to_string(),
-        intent_msg: IntentMessage::new(Intent::iota_transaction(), tx_data),
-        sender: IotaAddress::from_str("0x769364b73418667a44d600a8f0bc2d6655ef11cefb9e337fbd5aad5d4cc0552d").unwrap()
-    };
-    let sig = GenericSignature::PasskeyAuthenticator(
-        PasskeyAuthenticator::new_for_testing(
-            response.authenticator_data,
-            response.client_data_json,
-            Signature::from_bytes(&response.user_sig_bytes).unwrap(),
-        )
-        .unwrap(),
-    );
-
-    let res = sig.verify_authenticator(
-        &response.intent_msg,
-        response.sender,
-        0,
-        &Default::default(),
-        Arc::new(VerifiedDigestCache::new_empty()),
-    );
-    let err = res.unwrap_err();
-    assert_eq!(
-        err,
-        IotaError::InvalidSignature {
-            error: "Fails to verify".to_string()
-        }
-    );
-}
-
-#[tokio::test]
-async fn test_real_passkey_output() {
-    // response from a real passkey authenticator created in iCloud, from typescript client: https://github.com/joyqvq/iota-webauthn-poc/tree/joy/tx-example
-    let address =
-        IotaAddress::from_str("0xac8564f638fbf673fc92eb85b5abe5f7c29bdaa60a4a10329868fbe6c551dda2")
-            .unwrap();
-    let sig = GenericSignature::from_bytes(&Base64::decode("BiVJlg3liA6MaHQ0Fw9kdmBbj+SuuaKGMseZXPO6gx2XYx0AAAAAigF7InR5cGUiOiJ3ZWJhdXRobi5nZXQiLCJjaGFsbGVuZ2UiOiJBQUFBdF9taklCMXZiVnBZTTZXVjZZX29peDZKOGFOXzlzYjhTS0ZidWtCZmlRdyIsIm9yaWdpbiI6Imh0dHA6Ly9sb2NhbGhvc3Q6NTE3MyIsImNyb3NzT3JpZ2luIjpmYWxzZX1iApjskL9Xyfopyg9Av7MSrcchSpfWqAYoJ+qfSId4gNmoQ1YNgj2alDpRIbq9kthmyGY25+k24FrW114PEoy5C+8DPRcOCTtACi3ZywtZ4UILhwV+Suh79rWtbKqDqhBQwxM=").unwrap()).unwrap();
-    let tx_data: TransactionData = bcs::from_bytes(&Base64::decode("AAAAAKyFZPY4+/Zz/JLrhbWr5ffCm9qmCkoQMpho++bFUd2iAUwOMmeNHuxq2hS4PvO1uivs9exQGefW2wNQAt7tRkkdAgAAAAAAAAAgCsJHAaWbb8oUlZsGdsyW3Atf3d51wBEr9HLkrBF0/UushWT2OPv2c/yS64W1q+X3wpvapgpKEDKYaPvmxVHdougDAAAAAAAAgIQeAAAAAAAA").unwrap()).unwrap();
-    let res = sig.verify_authenticator(
-        &IntentMessage::new(Intent::iota_transaction(), tx_data),
-        address,
-        0,
-        &Default::default(),
-        Arc::new(VerifiedDigestCache::new_empty()),
-    );
-    assert!(res.is_ok());
-}
+// #[tokio::test]
+// async fn test_passkey_fails_not_normalized_signature() {
```

Comment:
Why is this and the next test commented?

Reply:
Removed tests for now, will re-enable them in a following PR.